src/main/java/teammates/ui/controller/CreateImageUploadUrlAjaxPageData.java

```diff
@@ -2,12 +2,14 @@
 
 import teammates.common.datatransfer.AccountAttributes;
 
-public class AdminEmailCreateImageUploadUrlAjaxPageData extends PageData {
+/**
+ * Page data for created image URL
```

Comment:
@damithc , could you give an example for this one ? "Page data for a page with created image URL" ?